plugins/tcpinfo/tcpinfo.cc

```diff
@@ -59,10 +59,14 @@ static const char *tcpi_headers[] = {
 
 struct Config {
   int sample;
-  unsigned log_level;
+  unsigned int log_level = 1;
+  unsigned int rolling_enabled = 1;
+  unsigned int rolling_interval_sec = 86400;
+  unsigned int rolling_offset_hr = 0;
+  unsigned int rolling_size_mb = 1024;
```

Comment:
You don't need to make these settings globally available. Just keep local variables in `TSPluginInit` and set them when you have created the log file.